app/src/main/java/woowacourse/shopping/data/payment/repository/CouponsRemoteDataSourceImpl.kt

```
@@ -0,0 +1,40 @@
+            .Builder()
+            .baseUrl(baseUrl)
+            .client(client)
+            .addConverterFactory(Json.asConverterFactory("application/json".toMediaType()))
+            .build()
+            .create(RetrofitService::class.java)
+
+    override suspend fun fetchCoupons(): CouponFetchResult<CouponResponse> =
+        try {
+            val response = retrofitService.requestCoupons()
+            CouponFetchResult.Success(response)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)